content/en/monitors/status/events.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Status Events
+further_reading:
+- link: "service_management/events/"
+  tag: "Documentation"
+  text: "Event Management"
+---
+
+<div class="alert alert-info">Status Events is part of the <a href="monitors/status/status_page">provisional Monitor Status Page</a>. If you are using the legacy status page, see the <a href="/monitors/status/status_legacy">Status Page (Legacy)</a> documentation.</div>
+
+## Overview
+
+{{< img src="/monitors/status/status_page_event_details.png" alt="Monitor status page displaying event details" style="width:100%;" >}}
+
+All events generated by your monitor appear on the status page, showing the groups' name, event type, and timestamp. The Event timeline also includes downtime and audit trail events.
+
+For each event, you can access Quick actions and view related assets, like dashboards and logs.
+
+## Event details section
+
+To explore each individual event for more information, including associated tags and action:
+
+1. From the monitor status page, scroll down to the **Event timeline.**   
+2. Click on an event in the timeline to view event details.  
+   
+Use the event details to understand monitor alerts and identify root causes. This information supports responder workflows and helps you stay informed about ongoing situations. 
+
+### Take action to remediate
+
+With Quick Actions, you can take action without leaving the Status page. Responders save time since the context is automatically added.
+
+| Action | Description |
+| :---- | :---- |
+| Mute  | Create a [downtime][3] to mute monitor alerts. |
+| Resolve | Temporarily set the monitor status to 'OK' for its next evaluation |
+| Declare Incident | Escalate monitor alerts with [Incident Management][4]. |
+| Create Case | Create a [case][5] if this needs further investigation and you want to correlate the alert to other Datadog products. |
+| Run Workflow | Run [Workflow][6] Automation with predefined snippets to run mitigation actions. |
+| Start a Collaboration | Start a [CoScreen][7] meeting and collaborate with your team. |
+
+## Event troubleshooting section
```

Comment:
@SimoneT-DD do you have an image of this or can you link to examples of this section?